### PR TITLE
[IMP]10707 add module accounting_report_improvement

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: accounting_report_improvement/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline accounting_report_improvement/__init__.py]: headers+README+ key etc.
name should be account_report_cost

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991 @seb-elico

**Comment by elicoidal** [comment]
LGTM

**Comment by siyuancn** [comment]
@Rona111 The code is too old and contains issue. Please check the baby project and get the new code.
The module name is changed to : accounting_report_cost_field

https://github.com/Elico-Corp/baby_international_odoo/pull/17
https://github.com/Elico-Corp/baby_international_odoo/pull/20

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn

**Comment by siyuancn** [comment]
LGTM

File: accounting_report_cost_field/LICENSE [hunk 2]
```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (elicoidal) [inline accounting_report_cost_field/LICENSE:1]: Actually small details: no need licence file

File: accounting_report_cost_field/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+accounting_report_cost_field
```
Comment (elicoidal) [inline accounting_report_cost_field/README.rst:6]: Accounting report with Cost field

**Pull request merged**